app/views/play/level/PlayLevelView.coffee

```diff
@@ -53,6 +53,7 @@ WebSurfaceView = require './WebSurfaceView'
 SpellPaletteView = require './tome/SpellPaletteView'
 
 require 'lib/game-libraries'
+window.Box2D = require('exports-loader?Box2D!vendor/scripts/Box2dWeb-2.1.a.3')
```

Comment:
@phoenixeliot Is this a problem for webpacking? Needed it for when we simulate game dev levels on the main thread.